logic/path/src/selectors/dfs.rs

```diff
@@ -174,17 +179,31 @@ impl<CW: EdgeWeighting<U256>> DfsPathSelector<CW> {
         }
 
         // Only use nodes that have shown to be somewhat reliable
-        if next_hop.quality < self.cfg.quality_threshold {
+        if next_hop.node_score < self.cfg.node_score_threshold {
             trace!(%next_hop, "node quality threshold not satisfied");
```

Comment:
Maybe consider adding a trace! level instrumentation to the entire function iwth the `next_hop` argument. Then it can be dropped from all traces thereafter.